src/appwindow.cpp

```diff
@@ -1010,6 +1010,12 @@ void AppWindow::onSettingsApplied(const QString &pagePath)
 {
     LOG_INFO("Apply settings for " << INFO_OF(pagePath));
 
+    for (int i = 0; i < ui->tabWidget->count(); ++i)
```

Comment:
Why do you want to apply the settings to each tab then to whole application?

Reply:
Because there's no need to do the reverse, which could lead to unforeseen bugs.

Reply:
I changed the order because of some issues, I'll check if it happens after your palette based stylings and let you know.

Reply:
I checked its fine!